scripts/tasks/src/presets.ts

```diff
@@ -119,6 +143,8 @@ export function preset() {
 
   task('build', series('clean', 'copy', 'sass', 'ts', 'api-extractor')).cached!();
 
+  task('build:react-components', series('clean', 'copy', 'sass', 'ts', 'swc:compile', 'api-extractor')).cached!();
```

Comment:
Did this initially since I was having issues with using `swc` for transpilation then having `babel` process the output. @Hotell was able to figure that out and this has now been changed with the updated code :)